src/types.ts

```diff
@@ -85,12 +85,18 @@ export type TopUpRawResponse = {
     quotedPaymentAmount: number;
     winstonCreditAmount: string;
   };
-  paymentSession: { url: string };
+  paymentSession: {
+    url: string | null;
+    id: string;
+    client_secret: string | null;
```

Comment:
Anything leaky or risky about this?

Reply:
`client_secret` is the same variable included appended to the end of the `url`